pcsx2-qt/AutoUpdaterDialog.cpp

```diff
@@ -455,6 +455,8 @@ void AutoUpdaterDialog::downloadUpdateClicked()
 	{
 		QMessageBox msgbox;
 		msgbox.setIcon(QMessageBox::Critical);
+		msgbox.setWindowModality(Qt::WindowModal);
```

Comment:
I think the save state warning should probably be application modal, and block all input, increasing the chances of the user actually reading it.

(application modal should be the default)